pyproject.toml

```
@@ -0,0 +1,24 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "hist-smooth"
+version = "0"
+description = "A great package."
+readme = "README.md"
+
+authors = [
+  { name = "Roman", email = "petrov.romash@gmail.com" },
+]
+requires-python = ">=3.9"
+
+dependencies = [
+  "hist[plot,fit]",

```

```suggestion
  "hist[plot,fit]>=2.8.0",
```

It is useful to put lower bounds (but not upper bounds) on library dependencies so that when installed with other tools the dependency resolver of the installer doesn't try to go to a much older version. Placing lower bounds helps the resolvers find compatible solves faster.

---

Resolved